docs/pages/components/alert.md

```diff
@@ -76,20 +84,72 @@ Add the modifier class `fd-alert--error` for Error alert.
 {% capture error-alert %}
 <div class="fd-alert fd-alert--error fd-alert--dismissible" role="alert" id="SsoiW591">
   <button class="fd-alert__close" aria-controls="SsoiW591" aria-label="Close"></button>
-  Vivamus sagittis diam in vehicula lobortis sapien arcu mattis erat vel aliquet sem urna et.
+  <p class="fd-alert__text">
+    Vivamus sagittis diam in vehicula lobortis sapien arcu mattis erat vel aliquet sem urna et.
+  </p>
 </div>
 {% endcapture %}
 
 {% include display-component.html component=error-alert %}
 
+## No Icons
+
+Add the modifier class `fd-alert--no-icon` to render alert without any icon
+
+{% capture no-icon %}
+<div class="fd-alert fd-alert--information fd-alert--no-icon fd-alert--dismissible" 
+     role="alert" id="SsoiW591">
+  <button class="fd-alert__close" aria-controls="SsoiW591" aria-label="Close"></button>
+  <p class="fd-alert__text">
+    Vivamus sagittis diam in vehicula lobortis sapien arcu mattis erat vel aliquet sem urna et.
+  </p>
+</div>
+
+<div class="fd-alert fd-alert--success fd-alert--no-icon fd-alert--dismissible" 
+     role="alert" id="SsoiW591">
+  <button class="fd-alert__close" aria-controls="SsoiW591" aria-label="Close"></button>
+  <p class="fd-alert__text">
+    Vivamus sagittis diam in vehicula lobortis sapien arcu mattis erat vel aliquet sem urna et.
+  </p>
+</div>
+
+<div class="fd-alert fd-alert--warning fd-alert--no-icon fd-alert--dismissible" 
+     role="alert" id="SsoiW591">
+  <button class="fd-alert__close" aria-controls="SsoiW591" aria-label="Close"></button>
+  <p class="fd-alert__text">
+    Vivamus sagittis diam in vehicula lobortis sapien arcu mattis erat vel aliquet sem urna et.
+  </p>
+</div>
+
+<div class="fd-alert fd-alert--error fd-alert--no-icon fd-alert--dismissible" 
+     role="alert" id="SsoiW591">
+  <button class="fd-alert__close" aria-controls="SsoiW591" aria-label="Close"></button>
+  <p class="fd-alert__text">
+    Vivamus sagittis diam in vehicula lobortis sapien arcu mattis erat vel aliquet sem urna et.
+  </p>
+</div>
+{% endcapture %}
+
+{% include display-component.html component=no-icon class='no-icon' %}
+
+
+
 ## Right to left support
 If right to left text direction is provided, the dismiss button moves to left side.
 
 {% capture default-alert %}
 <div class="fd-alert fd-alert--error fd-alert--dismissible" role="alert" id="BQ6WP516" dir="rtl">
   <button class="fd-alert__close" aria-controls="BQ6WP516" aria-label="Close"></button>
+  <p class="fd-alert__text">
    حقيقة وأساس تلك السعادة البشرية، فلا أحد يرفض أو يكره أو يتجنب الشعور بالسعادة، ولكن بفضل هؤلاء الأشخاص الذين لا يدركون بأن السعادة لا بد أن نستشعرها بصورة أكثر عقلانيأسي.
+   </p>
 </div>
 {% endcapture %}
 
 {% include display-component.html component=default-alert %}
+
+<style>
+  .docs-component__no-icon .fd-alert--no-icon{
+    margin-bottom: 20px;
+  }
+</style>
```

Comment:
Maybe remove this and just put a `<br />` in between each type (like all the other docs components do?

Reply:
done